Analytics Engineer/code/angular.json

```diff
@@ -30,11 +30,14 @@
             "styles": [
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
               "src/styles.scss"
-            ],
-            "scripts": []
+            ]
           },
           "configurations": {
             "production": {
+              "scripts": [
```

Comment:
I added this to only run the google analytics scripts in a production environment as to not skew tracking for devs. Another solution would be to dynamically update the analytics ID depending on the environment if you wanted to see analytics for something like staging.

Reply:
gtag is the newer version of the deprecated ga.js script that google is pushing developers to use.